index.d.ts

```diff
@@ -51,6 +51,35 @@ declare namespace prettyBytes {
 		```
 		*/
 		readonly binary?: boolean;
+
+		/**
+		The minimum number of fraction digits to display.
```

Comment:
It should say what the default behavior is (when not specified)

Reply:
Done.